### DISCUSSION TITLE
`sbt bloopInstall` compiles all modules

### DISCUSSION THREAD

**Discussion by kpodsiad** [scalacenter/bloop#1666] [Q&A]
I noticed that sometimes `sbt bloopInstall` compiles all modules when invoked. On the contrary, running `bloopInstall` from an already running sbt instance doesn't do that. 

It is bug or intended behavior? I thought that in general generating bloop files shouldn't require compilation.

**Comment by tgodzik** [comment]
This is usually caused if the workspace needs source generation, which I think invokes compilation itself.
